docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,520 @@
+
+        ![Addon settings main](media/addon_settings_main.png)
+
+    3.  Now, you can run BasicUI Web application by following the [instruction](../mobile/first-app.md#emulator)
+        
+        > [!NOTE]
+        > You will find nothing different from running the Web application without the addon because there's no visible change in UI. You will make a change on UI later.
+
+For more information on using the emulator features, see [Using Emulator Control Keys, Menu, and Panel](../../../tizen-studio/common-tools/emulator-control-panel.md) and [Using Extended Emulator Features](../../../tizen-studio/common-tools/emulator-features.md).
+
+<a name="target"></a>
+### Running on target device
+
+To run the application on a target device, you can follow the steps similar for the emulator. For more information, see [running BasicUI Web application in a target device](../mobile/first-app.md#target).
+
+In the target device, a BasicUI will appear as expected.
+
+
+<a name="ui"></a>
+## Designing simple UI
+
+The reason that there is no change even after installing the addon above, is that a callback function registered to addon event has only code to print out a log to console.
+
+Now, you can make some changes which is viewable in a Web application.
+
+You need to be familiar with the [basic knowledge in addons](../../guides/applications/addon.md).
+
+### Showing a pop-up on resume
+
+In this section, the example explains how to get the timing of resuming event in the Web application. On the timinig, a pop-up with `Hello. Resume occurred!!` message appears.
+
+First, the JavaScript file `tmplDefault.js` at `shared/res/` is like below:
+
+![JavaScript file for template](media/addon_template_js_file.png)
+
+By now, you must be familiar with the following for addons. For more information, see [basic knowledge in addons](../../guides/applications/addon.md).
+
+| Used Event     | Matched Callback    |
+| -------------- | ------------------- |
+| lcPrelaunch | prelaunch |
+| lcResume | resume |
+| lcSuspend | suspend |
+| lcQuit | quit |
+
+
+Each callback has only `console.log()` as implementation.
+
+Now, focus on only `lcResume` to show up a pop-up with some message. You can remove all the lines related to other events except `lcResume`.
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+You need to use `addonapi`, which includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`. For example:
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+    
+
+After using addonapi and adding the code, the JavaScript file looks like:
+
+![Code for resume pop-up](media/hello_resume_code.png)
+
+Steps you need to follow with the code are below:
```

Comment:
This line is ambigious. Can you please explain more?

Reply:
Changed as below:

Following are the steps to confirm if the changed file works properly: